lib/loggingService/LogEntry.js

```diff
@@ -9,8 +9,63 @@ var rootPath = process.cwd();
 var IncomingMessage = require('http').IncomingMessage;
 var ServerResponse = require('http').ServerResponse;
 
+function serializeError(error) {
+	// the stack trace and other error information will be moved into the data object
+
+	// turn the stack string into an array, stripped from its noisy whitespace
+
+	var stack = error.stack && error.stack.split(/\s*\n\s*at\s+/);
+	if (!stack || stack.length <= 1) {
+		return error;
+	}
+
+	// remove the first line: "Error: name", leaving only stack frames
+
+	stack.shift();
+
+	// assign all the data to the data object (custom errors may have some)
+	var data = Object.assign({
+		code: error.code,
+		type: error.name,
+		message: error.message || undefined
```

Comment:
You might as well assign to an empty object then, since you're copying over all properties.
However, if you want to keep it this way (to ensure these properties *always* exist), I would suggest removing `code`. It's not a standard (a convention at best).